sandboxes/bar/src/components/barChart.vue

```diff
@@ -0,0 +1,46 @@
+#Do not include the template tag in your .vue single-file components. Vue can
+#not merge templates. If you add an empty template tag, Vue will take the
+#template from your component and not from the extended one, which will result
+#in an empty template and unexpected errors.
+
+<script>
+import { Bar } from 'vue-chartjs'
+
+export default {
+  name: 'BarChart',
+  extends: Bar,
+  data: () => ({
+    chartdata: {
+      labels: [
+        'January',
+        'February',
+        'March',
+        'April',
+        'May',
+        'June',
+        'July',
+        'August',
+        'September',
+        'October',
+        'November',
+        'December'
+      ],
+      datasets: [
+        {
+          label: 'Data One',
+          backgroundColor: '#f87979',
+          data: [40, 20, 12, 39, 10, 40, 39, 80, 40, 20, 12, 11]
+        }
+      ]
+    },
+    options: {
```

Comment:
I know there are just examples, but I think some options/configs for chart we can freeze and put to computed

Reply:
Evan [said](https://twitter.com/phiterf/status/1408039716524539910?s=12) it was overhead. I agree with him.